modules/update-service-install-cli.adoc

```diff
@@ -0,0 +1,121 @@
+[id="update-service-install-cli_{context}"]
+= Installing the {product-title} Update Service by using the CLI
+
+You can use the {product-title} CLI to install the {product-title} Update Service Operator.
+
+.Procedure
+
+To install the {product-title} Update Service Operator by using the {product-title} CLI:
+
+. Create a Namespace for the {product-title} Update Service Operator:
+
+.. Create a Namespace object YAML file, for example, `update-service-namespace.yaml`, for the {product-title} Update Service Operator:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: FIXME-default-namespace
```

Comment:
This should probably be [`openshift-updateservice`][1].

[1]: https://github.com/openshift/cincinnati-operator/blob/4dd95332da8a79f8516255ebb54af0c84baffe72/config/olm-catalog/updateservice-operator/1.0.0/updateservice-operator.v1.0.0.clusterserviceversion.yaml#L23